core/client-internal/src/test/java/tachyon/client/file/FileInStreamTest.java

```diff
@@ -67,6 +67,7 @@
   private TachyonBlockStore mBlockStore;
   private FileSystemContext mContext;
   private FileInfo mInfo;
```

Comment:
Unfortunately not because we mutate the object sometimes which requires mutating `mInfo` and generating a new `URIStatus`. One way around it would be to use whitebox and modify `mStatus`'s internal state, but that seemed messier.